### PR TITLE
Feature: Generate and clean Dextrose stories

### PR DESCRIPTION
**Pull request by L0wry**

### CODE REVIEW

File: bin/run.js [hunk 1]
```diff
@@ -29,51 +31,68 @@ program
...
+  .command('clean-stories [path]')
+  .alias('cs')
+  .action((dirPath) => {
+    if (!dirPath) log.error('run', 'no directory path found ');
+
+    log.info('run', 'cleaning Dextrose stories');
+    cleanDextroseStories(dirPath);
+  });
+
+
```
Comment (milesillsley) [inline bin/run.js]: space!

File: bin/generate-stories.js [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+const overRideStories = (dirPath, files, stringToFind, stringToReplace) =>
+  new Promise((resolve, reject) => {
+    files.forEach((file) => {
+      fs.readFile(file, 'utf8', (err, data) => {
+        if (err) {
+          reject(err);
+        }
+        if (data.includes(stringToFind)) {
+          const updatedData = data.replace(stringToFind, stringToReplace);
+          const trimmedFile = file.replace('.js', '');
```
Comment (asifhafeez) [inline bin/generate-stories.js]: can't you use this instead and get rid of the next line?

```const trimmedFile = file.replace('.js', dextroseExtension)```

File: bin/generate-stories.js [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+          fs.writeFileSync(`${dirPath}/${newFilePath}`, updatedData);
+        }
+      });
+    });
+    resolve();
+  });
+
+const findStoryFiles = (dir, matchers) => new Promise((resolve, reject) => {
+  fs.readdir(dir, 'utf8', (err, files) => {
+    if (err) reject(err);
```
Comment (asifhafeez) [inline bin/generate-stories.js]: Just to check, I think @toboid said we shouldn't be using one line if statements? Does this not apply on the dextrose app?

File: bin/generate-stories.js [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+    if (!files) {
+      reject(Error('No files found'));
+    } else {
+      const storyFiles = files.map((file) => {
+        const isStoryFile = matchers.some(matcher => file.includes(matcher));
+        if (isStoryFile) {
+          return file;
+        }
+
+        return undefined;
```
Comment (asifhafeez) [inline bin/generate-stories.js]: I don't think you need to return undefined, it should return undefined automatically.

File: bin/generate-stories.js (hunk 3 above)
Comment (L0wry) [reply bin/generate-stories.js]: dextrose is new and cool we use this a couple times throughout

File: bin/generate-stories.js (hunk 2 above)
Comment (L0wry) [reply bin/generate-stories.js]: 👍

File: bin/run.js (hunk 1 above)
Comment (L0wry) [reply bin/run.js]: 👍

File: bin/generate-stories.js (hunk 4 above)
Comment (L0wry) [reply bin/generate-stories.js]: this is something that lint made me do :(

**Pull request merged**